docker-compose.yml

```diff
@@ -170,26 +158,29 @@ services:
       STORE_PGDATABASE: dev_database
 
       ADMIN_API_PORT: 3001
-      STORE_API: http://store-api:3000
+      STORE_API: http://stpre-api:3000
+      LOCAL_CORS: true
```

Comment:
This cannot be hardcoded like this. There may be deployments using this docker-compose.yml with an nginx ontop, in which case specifying `LOCAL_CORS` is flawed and will result in a broken deployment.